BodyPart3d[] body_parts
uint32 id
